--- traefik_repo/pkg/config/static/static_config.go	2022-03-09 18:28:54.743867530 +0100
+++ traefik_repo/pkg/config/static/static_config_b.go	2022-03-09 18:29:08.423661056 +0100
@@ -26,6 +26,7 @@
	"github.com/traefik/traefik/v3/pkg/provider/kv/etcd"
 	"github.com/traefik/traefik/v3/pkg/provider/kv/redis"
 	"github.com/traefik/traefik/v3/pkg/provider/kv/zk"
+	"github.com/traefik/traefik/v3/pkg/provider/mesos"
	"github.com/traefik/traefik/v3/pkg/provider/nomad"
 	"github.com/traefik/traefik/v2/pkg/provider/rest"
	"github.com/traefik/traefik/v3/pkg/tracing/opentelemetry"	
@@ -236,6 +236,7 @@
	File                   *file.Provider                 `description:"Enables File provider." json:"file,omitempty" toml:"file,omitempty" yaml:"file,omitempty" export:"true"`
+	Mesos                  *mesos.Provider                `description:"Enable Mesos backend with default settings." json:"mesos,omitempty" toml:"mesos,omitempty" yaml:"mesos,omitempty" export:"true" label:"allowEmpty" file:"allowEmpty"`
	KubernetesIngress      *ingress.Provider              `description:"Enables Kubernetes Ingress provider." json:"kubernetesIngress,omitempty" toml:"kubernetesIngress,omitempty" yaml:"kubernetesIngress,omitempty" label:"allowEmpty" file:"allowEmpty" export:"true"`
	KubernetesIngressNGINX *ingressnginx.Provider         `description:"Enables Kubernetes Ingress NGINX provider." json:"kubernetesIngressNGINX,omitempty" toml:"kubernetesIngressNGINX,omitempty" yaml:"kubernetesIngressNGINX,omitempty" label:"allowEmpty" file:"allowEmpty" export:"true"`
	KubernetesCRD          *crd.Provider                  `description:"Enables Kubernetes CRD provider." json:"kubernetesCRD,omitempty" toml:"kubernetesCRD,omitempty" yaml:"kubernetesCRD,omitempty" label:"allowEmpty" file:"allowEmpty" export:"true"`
	KubernetesGateway      *gateway.Provider              `description:"Enables Kubernetes Gateway API provider." json:"kubernetesGateway,omitempty" toml:"kubernetesGateway,omitempty" yaml:"kubernetesGateway,omitempty" label:"allowEmpty" file:"allowEmpty" export:"true"`
